docs/tools-release-process.md

```diff
@@ -32,6 +32,19 @@ A tools release is performed by a single person each week. We call this person t
 ## Decide on the next version numbers
 According [versioning-and-release-strategy.md](versioning-and-release-strategy.md) patch version bumps (the last of 3 numbers) should only be used for fixes and updates of references to platform versions. For any change in functionality, the second (minor) part of the version should be bumped and new branch created. Instructions for creating a new release branch are further down on this page.
 
+## Tools release planning
+
+### cordova-create principle
```

Comment:
Should it say cordova-common here?

Reply:
Yes or maybe something else. TBH I remember adding this header in order to balance with the "Alternative approaches" header. I am working on some more general comments on this PR.